### PR TITLE
Single line install

### PR DESCRIPTION
**Pull request by oudeismetis**
- [x] Test indivual steps of new commands locally
- [x] Test new install steps in README on a live Raspberry Pi

### CODE REVIEW

File: Contribute.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Dev Install
```
Comment (cherryshoe) [inline Contribute.md]: Looks like standard filename is contributing.md. https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/setting-guidelines-for-repository-contributors

File: README.md [hunk 2]
```diff
@@ -4,85 +4,64 @@ A running camera feed that only saves videos when you press a button. How many t
...
-    ```
-        chmod a+x install.sh
-        ./install.sh
-    ```
-1. Remember to do as the install script says: Logout and login again for the audio server to have configuration recognized - no need to reboot/restart
-1. missed-moment is ready for use, push the button to save a moment!
-1. missed-moment-usb is ready for use, grab videos to a USB
+## Contributing
+
+Details on getting setup via git and contributing back can be found [Here](Contribute.md)
```
Comment (cherryshoe) [inline README.md]: Related to my other comment, name of contributing file

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+1. Make sure you've enabled the camera module in the settings
+1. Install GPAC to do video format operations and exfat-fuse for USB support
+
+    ```
+        sudo apt update
+        sudo apt install gpac
+        sudo apt install exfat-fuse
+    ```
+
+1. Install Python libraries
```
Comment (cherryshoe) [inline contributing.md:21]: There's things missing now that I forgot to add to the original README.  If you look at install.sh it needs audio server library and conf setup.

Comment (oudeismetis) [reply contributing.md:21]: Yes we need to fix that, but it's out of scope for this PR.

**Review by cherryshoe — APPROVED** [review]

**Pull request merged**